src/client/zweb/deployer/index.js

```diff
@@ -427,13 +491,12 @@ class Deployer {
                             // do not found metadata file or
                             // forcing the deployment of a new proxy
                             if (
-                                proxyAddress == null || 
-                                proxyDescriptionFileId == null || 
+                                !proxyAddress ||
+                                !proxyDescriptionFileId ||
                                 force_deploy_proxy 
                             ) {
                                 //deploy a new proxy
                                 log.debug('deployProxy call');
-                                const cfg = {kind: 'uups'};
```

Comment:
What is the reason for removing this config?

Reply:
It's not used

Reply:
@Koochr it was used before your change.